packages/phoenix-ng/projects/phoenix-app/src/app/sections/lhcb/lhcb.component.ts

```diff
@@ -52,9 +52,100 @@ export class LHCbComponent implements OnInit {
 
     this.eventDisplay.init(configuration);
 
+    /** 
+     * The following commented part of code is properly working
+     * and it is the whole new Geometry of the run3 LHCb detector in one single gltf file,
+     * but there is a huge performance issue (max 4fps)
+     * the fault is the root file that was designed with too many subchildren for the parts of
+     * Ecal, Hcal, Muon
+     * and thus the reuslted gltf file generated from the root one has huge performance issues. 
+     * Uncomment the following to see it in action ( you'll have to wait a bit :) )
+    */
+
+    // this.eventDisplay.loadGLTFGeometry(
```

Comment:
Why would you want this if you have the sub-geometries  though?

Reply:
Yeah, we don't need it, I just left it there for demonstration purposes of the performance that I wanted to do in today's meeting, but then I forgot to remove it from the pull request, thank you for reminding, I'll do it now! :)